### PR TITLE
a more in depth doc on cacheable option

### PR DESCRIPTION
**Pull request by bcoca**
##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

set_fact
##### ANSIBLE VERSION

```paste below
2.4+
```

### CODE REVIEW

File: lib/ansible/modules/utilities/logic/set_fact.py [hunk 1]
```diff
@@ -35,8 +35,9 @@
     required: true
   cacheable:
     description:
-      - This boolean indicates if the facts set will also be added to the
-        fact cache, if fact caching is enabled.
+      - This boolean converts the variable into and actual 'fact' which will also be added to the fact cache, if fact caching is enabled.
```
Comment (jborean93) [inline lib/ansible/modules/utilities/logic/set_fact.py]: s/and/an/

File: lib/ansible/modules/utilities/logic/set_fact.py [hunk 2]
```diff
@@ -35,8 +35,9 @@
...
+      - Normally this module creates 'host level variables' and has much higher precedence, this option changes the nature and precedence
```
Comment (jborean93) [inline lib/ansible/modules/utilities/logic/set_fact.py:39]: Are you able to state what that precedence is in terms of the ones at https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable (maybe just a link).

Comment (bcoca) [reply lib/ansible/modules/utilities/logic/set_fact.py:39]: well, that list already states 'set_fact' vs 'host fact' are 7 steps away

Comment (jborean93) [reply lib/ansible/modules/utilities/logic/set_fact.py:39]: Fair enough, should have actually gone to that link and read it properly

**Review by jborean93 — APPROVED** [review]

**Pull request merged**